app/controllers/api/skater_profiles_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::SkaterProfilesController < ApplicationController
+  def show
+    profile = SkaterProfile.find_by(skater_id: current_skater.id)
+
+    render json: profile
+  end
+
+  def create
+    profile = SkaterProfile.new(create_params)
+    profile.save!
```

Comment:
Instead of calling .new and .save!, you can just call create!

profile = SkaterProfile.create!(create_params)